pkg/cmd/server/api/validation/master.go

```diff
@@ -287,6 +287,18 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	if len(config.LoggingPublicURL) > 0 {
+		if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
```

Comment:
Yes, logging and metrics endpoints should be https. It would be a mistake
for it to be otherwise, so should be validated.

On Tue, Oct 6, 2015 at 11:14 AM, Sam Padgett <notifications@github.com>
wrote:

> In pkg/cmd/server/api/validation/master.go
> <https://github.com/openshift/origin/pull/4954#discussion_r41277461>:
>
> > @@ -287,6 +287,18 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
> >  		allErrs = append(allErrs, urlErrs...)
> >  	}
> >
> > +	if len(config.LoggingPublicURL) > 0 {
> > +		if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
>
> @sosiouxme <https://github.com/sosiouxme> Does the logging endpoint take
> an API token?
>
> These probably need to be https anyway to avoid mixed content problems in
> the browser.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4954/files#r41277461>.
>